src/popup/snooze.scss

```diff
@@ -652,7 +652,7 @@ li.entry {
     border-radius: 2px;
     font-size: 14px;
     text-align: center;
-    width: 6em;
+    width: 7.85em;
```

Comment:
This seems pretty hacky... My CSS-fu is failing me in getting the widths on the time selector right.